lib/simplify.dart

```diff
@@ -0,0 +1,121 @@
+// implementation based on
```

Comment:
Still not sure about this, I feel like it's more polluting than it was. Definitely shouldn't be under 'lib/'. Would be good if @ignatz and the rest of the maintainers could chime in here.

Reply:
@ibrierley @TesteurManiak @josxha looking for some feedback!

Reply:
I agree with @JaffaKetchup, I'm not sure that we need to expose to the users. IMO, this file would be better in the `lib/src/misc/` for internal use only 🤔

Reply:
(Just to say, I was originally for exposing it, just not quite in this way. But I do now agree that maybe keeping it private makes sense.)